DrawAction/DrawDivide.swift

```diff
@@ -67,19 +67,17 @@ final public class DrawDivide : DrawAction {
      */
     public convenience init(amount: CGFloat, padding: CGFloat, edge: CGRectEdge, slice: DrawAction?, next: DrawAction) {
         self.init(amount: amount, padding: padding, edge: edge, slice: slice)
-        self.add(next)
+        let _ = self.add(next)
```

Comment:
Instead of doing this, I think we should mark 'add' with `@discardableResult`, since it only returns self for chaining purposes, it will often be dropped.